ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/tabsontop/v1/tabsontop/tabsontop.html

```diff
@@ -36,7 +36,7 @@
      <div data-sly-call="${longDescription.longDescription @componentId=panel.id, longDescription=panel.description, bemBlock='cmp-tabs'}" data-sly-unwrap></div>
 
  <ol data-sly-test="${tabs.items && tabs.items.size > 0}"
-     data-sly-list.tab="${tabs.items}"
+     data-sly-list.tab="${panel.items}"
```

Comment:
This works without updating panel sling model? If so do we still need to initialise tab sling model?

Reply:
tabs.accessibilityLabel and tabs.activeItem are coming from tabs, so we still needs tabs sling model.